models/demos/wormhole/llama31_8b/tests/test_llama_model.py

```diff
@@ -29,42 +29,37 @@
 @pytest.mark.timeout(900)
 @pytest.mark.models_performance_bare_metal
 @pytest.mark.parametrize(
-    "iterations",
-    (17,),
+    "weights, layers",
+    [
+        ("random", 1),
+        ("general", 32),
+    ],
+    ids=["quick", "full"],
 )
-def test_llama_model_inference(device, iterations, use_program_cache, reset_seeds):
+def test_llama_model_inference(device, weights, layers, use_program_cache, reset_seeds):
     run_ref_pt = True  # Flag to run reference PyTorch model and compare PCC
-    cache_pcc = False  # Flag to measure KV cache PCC for all layers
+    cache_pcc = layers == 1  # Flag to measure KV cache PCC for all layers
 
     dtype = ttnn.bfloat8_b
-    pcc = 0.92  # FIXME: why are first couple of iterations 0.93 and the rest higher?
+    pcc = 0.9985 if layers == 1 else 0.97
+    iterations = 6 if layers == 1 else 9
```

Comment:
Is 6 iterations and 9 just to avoid the output to go crazy afterwards?
If so, should we add a comment on that?